docs/_future/manifest.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Manifest
+lang: en
+slug: manifest
+order: 2
+layout: future
+---
+
+
+<div class="section-content">
+
+Your project should contain a `manifest.json` file that defines your app's manifest, where you can configure your app. Refer to <a href="https://api.slack.com/reference/manifests" target="_blank">app manifest documentation</a> to learn about the available manifest configurations.
+
+Notably this file informs slack of the definitions for:
+
+* [functions](/bolt-python/future/concepts#manifest-functions)
+* [workflows](/bolt-python/future/concepts#manifest-workflows)
+
+`manifest.json` should be located at the top level of your project, along with a `triggers` folder containing `*_trigger.json` files that define the <a href="https://api.slack.com/future/triggers" target="_blank">triggers</a> for your app.
```

Comment:
Since bolt-python supports manifest.json outside of the project root, I'm not sure how valuable it is for devs to be too prescriptive here. I think we can remove this

Reply:
I included this in just to show where the manifest file is located in our templates, some of them have a lot configuration files at the top level of the project

@srajiang what do you think of just changing to wording like this
```suggestion
`manifest.json` should be located at the top level of our example projects, along with a `triggers` folder containing `*_trigger.json` files that define the <a href="https://api.slack.com/future/triggers" target="_blank">triggers</a> for your app.
```

Reply:
```suggestion
`manifest.json` is located at the top level of our example projects, along with a `triggers` folder containing `*_trigger.json` files that define the <a href="https://api.slack.com/future/triggers" target="_blank">triggers</a> for your app.
```
Ah okay, maybe let's go all the way ☝️ and link to the example project as well?

Reply:
Yess the link is a great idea 💯  Ill put the template link